mixins/FluxibleMixin.js

```diff
@@ -53,6 +57,18 @@ StoreMixin = {
     },
 
     /**
+     * Calls an action
+     * @method executeAction
+     */
+    executeAction: function executeAction() {
+        var context = this.props.context || this.context;
+        if (!context || !context.executeAction) {
+            throw new Error('executeAction was called but no context was provided');
```

Comment:
make the error message more helpful to users by telling them we expect it from this.props.context or this.context.